docs/index.md

```diff
@@ -1,131 +1,250 @@
-
-```py title="hello.py"
---8<-- "docs/source_files/pixi_workspaces/introduction/deps_add/hello.py"
-```
-
-The dependencies are installed in a Pixi environment.
-In order to run a command within an environment, we prefix it with `pixi run`.
-
-```bash
-pixi run python hello.py
-```
-
+Install your favorite tools with a single command.
+```shell
+pixi global install git gh nvim ipython btop bat starship ripgrep
 ```
- __________________
-< Hello Pixi fans! >
- ------------------
-     \   ^__^
-      \  (oo)\_______
-         (__)\       )\/\
-           ||----w |
-           ||     ||
+![Pixi Global Tools demo](assets/pixi_global_demo.gif)
 
-```
+---
 
 
-You can also put this run command in a **task**.
+## ↔️ What is the difference with Pixi?
 
-```bash
-pixi task add start python hello.py
-```
+| Builtin Core Features | Pixi | Conda | Pip | Poetry | uv |
+|-----------------------|---|---|---|---|---|
+| Installs Python | ✅ | ✅ | ❌ | ❌ | ✅ |
+| Supports more than Python | [✅]("Using the conda ecosystem Pixi installs any type of package, not just Python!") | ✅ | [❌]("Only Python packages") | [❌]("Only Python packages") |[❌]("Only Python packages") |
+| Cross-platform Task Runner | [✅](workspace/advanced_tasks.md "Run shell commands on all platforms with `pixi run`") | ❌ | ❌ | ❌ | ✅ |
+| Lockfiles | [✅](workspace/lockfile.md) | ❌ | ❌ | ✅ | ✅ |
+| Project Management | [✅](reference/pixi_manifest.md) | ❌ | ❌ | ✅ | ✅ |
 
-```toml title="pixi.toml" hl_lines="6-7"
---8<-- "docs/source_files/pixi_workspaces/introduction/task_add/pixi.toml"
-```
+---
 
-After adding the task, you can run the task using its name.
-
-```bash
-pixi run start
-```
 
-```
- __________________
-< Hello Pixi fans! >
- ------------------
-     \   ^__^
-      \  (oo)\_______
-         (__)\       )\/\
-           ||----w |
-           ||     ||
+## 🛠️ Installation
 
-```
+To install `pixi`, run:
 
+=== "Linux & macOS"
+    ```bash
+    curl -fsSL https://pixi.sh/install.sh | sh
+    ```
 
-You now know how to add dependencies and tasks to your environment.
-Put the workspace folder on a different machine, and you will find that Pixi will be able to fully reproduce your setup.
+=== "Windows"
+    ```powershell
+    powershell -ExecutionPolicy ByPass -c "irm -useb https://pixi.sh/install.ps1 | iex"
+    ```
 
-If you want to learn more about Pixi, check out the next page!
+!!! tip "Now restart your terminal or shell!"
+    The installation needs to become effective by restarting your terminal or sourcing your shell.
+
+??? question "Don't trust our link? Check the script!"
+    You can check the installation `sh` script: [download](https://pixi.sh/install.sh) and the `ps1`: [download](https://pixi.sh/install.ps1).
+    The scripts are open source and available on [GitHub](https://github.com/prefix-dev/pixi/tree/main/install).
+
+[**See all installation options →**](installation.md)
+
+---
+
+## 🏁 Getting Started
+
+=== "Python"
```

Comment:
i think another cross-platform dev-ops example is actually quite nice with pixi: you can install terraform/opentofu, ansible, kubectl, k9s, just/make, ... all with one environment, even if all the tools are coming from different ecosystems